k8s/helmfile/env/local/istio-service-mesh.values.yaml.gotmpl

```diff
@@ -0,0 +1,19 @@
+podMonitor:
+  namespace: monitoring
+
+serviceMonitor:
+  namespace: monitoring
+
+istiod:
+  pilot:
+    resources:
+      requests:
+        cpu: 125m
```

Comment:
I tried quickly to answer this myself but figured asking would be better:
Where did these numbers come from? Do you think they are right after some experimenting or are these some defaults from somewhere?

Reply:
Those are mostly educated guesses from what I've seen on my machine, however I obviously couldn't do this for staging or production.

The reason I did changes there is that the values seemed _very_ high to me (when comparing to what I've seen sth like nginx consume). They might be totally wrong though.